spec/namespaces/cluster.yaml

```diff
@@ -64,7 +64,7 @@ paths:
       operationId: cluster.put_decommission_awareness.0
       x-operation-group: cluster.put_decommission_awareness
       x-version-added: '1.0'
-      description: Decommissions an awareness attribute.
+      description: Decommissions a cluster zone based on awareness. It greatly benefits multi-zone deployments, where awareness attributes can aid in applying new upgrades to a cluster in a controlled fashion. 
```

Comment:
```suggestion
      description: Decommissions a cluster zone based on awareness. This can greatly benefit multi-zone deployments, where awareness attributes can aid in applying new upgrades to a cluster in a controlled fashion. 
```